common/v2/components/TransactionFlow/displays/FromToAccount.tsx

```diff
@@ -3,14 +3,18 @@ import styled from 'styled-components';
 
 import translate, { translateRaw } from 'v2/translations';
 
-import { StoreAccount } from 'v2/types';
 import { COLORS, BREAK_POINTS } from 'v2/theme';
 import { truncate } from 'v2/utils';
 import { Typography, Account } from 'v2/components';
 
+export interface IAddressAndLabel {
+  address: string;
+  label: string | undefined;
+}
+
 interface Props {
-  fromAccount: StoreAccount;
-  toAccount: StoreAccount;
+  fromAddressAndLabel: IAddressAndLabel;
+  toAddressAndLabel: IAddressAndLabel;
```

Comment:
IMHO we don't need to be verbose, especially since this is contained to this component.
Reading the type should be sufficient to know what it's composed of. This allows us to have a variable name that is more fluid to read.

```
interface Props = {
 from: Pick<StoreAccount, 'address' | 'label'>, 
 to: Pick<StoreAccount, 'address' | 'label'>
}
```